runtime/interpreter/value.go

```diff
@@ -10765,6 +10766,44 @@ func (PathValue) IsStorable() bool {
 	return true
 }
 
+func convertPath(domain common.PathDomain, value Value) Value {
+	stringValue, ok := value.(*StringValue)
+	if !ok {
+		return NilValue{}
+	}
+
+	_, err := sema.CheckPathLiteral(
+		"public",
+		stringValue.Str,
+		func() ast.Range {
+			return ReturnEmptyLocationRange().Range
+		},
+		func() ast.Range {
+			return ReturnEmptyLocationRange().Range
+		},
```

Comment:
No need to construct a full location range (location + range), just to get an empty range:

```suggestion
		func() ast.Range {
			return ast.Range{}
		},
		func() ast.Range {
			return ast.Range{}
		},
```

And then further, maybe create a function like `ReturnEmptyLocationRange`:

```go
func ReturnEmptyRange() ast.Range {
	return ast.Range{}
}
```

```suggestion
		ReturnEmptyRange,
		ReturnEmptyRange,
```